pkg/cmd/backuprestore/backupserver.go

```diff
@@ -42,11 +48,16 @@ type backupServer struct {
 	enabled      bool
 	cronSchedule cron.Schedule
 	backupOptions
+	prune.PruneOpts
 }
 
 func NewBackupServer(ctx context.Context) *cobra.Command {
 	backupSrv := &backupServer{
 		backupOptions: backupOptions{errOut: os.Stderr},
+		PruneOpts: prune.PruneOpts{
+			RetentionType: "None",
+			BackupPath:    backupVolume,
+		},
```

Comment:
do we need to set this?

Reply:
so the backup path is const [here](https://github.com/openshift/cluster-etcd-operator/blob/735f37b8d3938a03cdf1c74e61f634a8adcf97c2/pkg/cmd/prune-backups/prune.go#L18)

In order to user Prune, and use the NoConfig backup [volume](https://github.com/openshift/cluster-etcd-operator/blob/735f37b8d3938a03cdf1c74e61f634a8adcf97c2/pkg/cmd/backuprestore/backupserver.go#L18), I override the path here

please correct me if I am wrong

Reply:
but is this correct?
because you're setting the backupDir path to "backupVolume + dateString"

Reply:
nvm, that's just the suffix for each backup invocation

Reply:
genauuu :D :D